rootconf/default/etc/init.d/console-logger.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+#  Copyright (C) 2016 Audi Hsu <audi.hsu@quantatw.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+##
+## redirect console_log_file and tee_log_file to /dev/console
+##
+
+. /lib/onie/functions
+
+touch $console_log_file && /usr/bin/tail -f $console_log_file > /dev/console &
```

Comment:
Since _log_console_msg and other different log message levels, I shall keep the console_log_file and tee_log_file.
Unless the logging message definition is changed.